rewrite-groovy/src/test/java/org/openrewrite/java/cleanup/ChangeTypeTest.java

```diff
@@ -103,4 +105,31 @@ class newFile {
           )
         );
     }
+
+    @SuppressWarnings("DataFlowIssue")
+    @Disabled("fails because there's a reference change but no content diff but that's the point; would need to adjust RewriteTest")
```

Comment:
open to suggestions if there's an easy workaround for this or if we'd prefer to just delete the test

Reply:
While we wait for a real answer to your question you could already swap `@Disabled` for `@ExpectedToFail`. Then we at least know if this gets fixed accidentally. 
```suggestion
    @ExpectedToFail("fails because there's a reference change but no content diff but that's the point; would need to adjust RewriteTest")
```

Reply:
I didn't know about that annotation and I love it; will make that change in a few

Reply:
Sure! If you combine it with an `@Issue` link we have a feedback loop on where to post the update if fixed.

Reply:
done, with new issue ref https://github.com/openrewrite/rewrite/issues/3058